src/index.html

```diff
@@ -3,12 +3,280 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Dia</title>
+    <title>DIA</title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,600,700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Poppins:400,600,700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.css">
+    <link rel="icon" href="./images/logo/flower.png">
+    <script type="module" src="scripts/main.js"></script>
   </head>
   <body>
-    <img src="./images/logo.svg" alt="Logo">
-    <h1>Dia</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+    <header class="header"  id="home">
+      <div class="wrapper wrapper--header">
+        <button class="header__button"></button>
+        <div class="header__nav">
+          <a href="" class="header__link"><img src="./images/logo/Logo.svg" alt="logo"></a>
+          <nav class="nav">
+              <ul class="nav__list">
+                <li class="nav__item">
+                  <a href="" class="nav__link">Home</a>
+                </li>
+                <li class="nav__item">
+                  <a href="#what-do" class="nav__link">About us</a>
+                </li>
+                <li class="nav__item">
+                  <a href="#process" class="nav__link">Services</a>
+                </li>
+                <li class="nav__item">
+                  <a href="#features" class="nav__link">Work</a>
+                </li>
+                <li class="nav__item">
+                  <a href="#contact" class="nav__link">Contact Us</a>
+                </li>
+              </ul>
+              <a href="#vision" class="nav__link nav__link--is-active">Here us</a>
+          </nav>
+        </div>
+
+
+        <div class="epigraph">
+          <h1 class="epigraph__title">Global digital agency</h1>
+          <p class="epigraph__text">By the same illusion which lifts the horizon of the sea to the level of the spectator on a hillside, the sable cloud beneath.</p>
```

Comment:
переноси длинные строки
```suggestion
          <p class="epigraph__text">
            By the same illusion which lifts the horizon of the sea to the level of the spectator on a hillside, the sable cloud beneath.
          </p>
```